charts/osm/values.yaml

```diff
@@ -12,19 +12,19 @@ OpenServiceMesh:
     # -- Container image pull policy for control plane containers
     pullPolicy: IfNotPresent
     # -- Container image tag for control plane images
-    tag: "latest-main"
+    tag: "v0.11.0-rc.1"
```

Comment:
We want to be setting this tag only from the CI job for the PR workflow, otherwise, the default images will also use tags instead of digests.